### PR TITLE
Added Sonesta Brands (first edit)

### PR DESCRIPTION
**Pull request by arnowrld**
This is my first edit to the nsi. Could use some feedback on improvement.

### CODE REVIEW

File: data/brands/tourism/hotel.json [hunk 1]
```diff
@@ -1961,6 +1961,28 @@
         "tourism": "hotel"
       }
     },
+    {
+      "displayName": "Sonesta Select",
+      "id": "sonestaselect-u76bs5"
```
Comment (arch0345) [inline data/brands/tourism/hotel.json:1966]: Missing comma
```suggestion
      "id": "sonestaselect-u76bs5",
```

File: data/brands/tourism/hotel.json [hunk 2]
```diff
@@ -1961,6 +1961,28 @@
...
+      "tags": {
+        "brand": "Sonesta Select",
+        "brand:wikidata": "Q109272530",
+        "name": "Sonesta Select",
+        "tourism": "hotel"
+      }
+    },
+    {
+      "displayName": "Sonesta Simply Suites",
+      "id": "sonestasimplysuites-m4ud59"
```
Comment (arch0345) [inline data/brands/tourism/hotel.json:1977]:
```suggestion
      "id": "sonestasimplysuites-m4ud59"
```

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks @arnowrld! There's just a few missing commas - you can click on `commit suggestion` to use my suggestions or edit it manually going to the `Files changed` tab.

**Comment by bhousel** [comment]
Thanks for the quick fixes @arnowrld and welcome to the project 👋

**Pull request merged**